.github/workflows/pr_checks.yml

```
@@ -0,0 +1,55 @@
+name: PR Checks
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)